_data/backlog.yml

```
@@ -1,9 +1,135 @@
 cards:
+  - title: Connection Prompting
+    description:
+      - Reintroduce the "Prompt Mode" which asks for your input whether or not to allow a connection. Naturally, make it possible to lock in a decision for the future.
+    projects:
+      - Portmaster
+  - title: Full UI Redesign
+    description:
+      - Completely overhaul the Portmaster user interface to simplify all aspects of it.
+      - Easily block connections from the Monitor, easily block all connection of an app, etc...
+    projects:
+      - Portmaster
+  - title: Network Trust System
+    description:
+      - Remember which networks you trust and what security level should be used within them.
+      - Breathe some life into the Autopilot system so a switch in networks automatically changes the security level.
```

Sentence is a bit rough, would suggest:
```suggestion
      - Breathe some life into the Autopilot system which will automatically switch to a heightened security level in untrusted networks.
```